src/components/date-picker/date-picker.tsx

```diff
@@ -120,22 +128,25 @@ export class DatePicker implements LocalizedComponent {
    * Localized string for "previous month" (used for aria label)
    *
    * @default "Previous month"
+   * @deprecated - translations are now built-in, if you need to override a string, please use `messageOverrides`
    */
-  @Prop() intlPrevMonth?: string = TEXT.prevMonth;
+  @Prop() intlPrevMonth?: string;
```

Comment:
IMO, we don't need to mark props as optional as Stencil will do this for us for the generated custom element type.

Reply:
marking it as optional helps us leverage documenting the prop as optional for the user.